enhance matching

aaronhurt opened the pull request:
This commit reworks matching. The new matcher ignores spacing and order differences from Helm. The format of the `Map.yaml` file has been simplified to reflect these changes. The README.md has been updated, and a new converter tool has been provided. The current `Map.yaml` in this PR was generated from the previous `Map.yaml` via the converter.

This PR is breaking in it's current state, it will only accept the new `Map.yaml` format, but as stated previously a converter has been provided.

robertsirc commented:
Hello, thank you for your PR, this branch has some conflicts could you please correct this and we can take a look.

aaronhurt committed:
feat: enhance matching

This commit reworks matching. The new matcher ignores spacing and order
differences from Helm. The format of the `Map.yaml` file has been
simplified to reflect these changes. The README.md has been updated,
and a new converter tool has been provided. The current `Map.yaml` in
this PR was generated from the previous `Map.yaml` via the converter.

Signed-off-by: Aaron Hurt <aaron.hurt@bayer.com>

aaronhurt commented:
Hi @robertsirc, sorry for the delay here. I've rebased my fork off the current main.

aaronhurt commented:
Updated github.com/containerd/containerd to v1.7.29 to resolve GO-2025-4108 and GO-2025-4100 as detected by govulncheck, and force pushed my branch.
